blog/poc3.md

```diff
@@ -0,0 +1,127 @@
+getting deleted. Storage insurance (to be implemented in POC4 2019) will offer users a secure guarantee to protect important content from being purged.
+
+Since for delivery, each node needs to know only the immediate peer upstream, there is no need to know who originated the request. As a consequence forwarding kademlia allows for "anonymous private browsing" as it were.
+
+The Swarm content service offers an HTTP interface via a local proxy running on your local machine and connected to the rest of the swarm. This proxy used in the browser to serve static assets (such as HTML and Javascript for dynamic content) of decentralised applications. The proxy offers six url schemes that dapps can use to interact with Swarm content storage API.
+A manifest is a data structure describing file collections; they specify paths and corresponding content hashes allowing for URL based content retrieval. The `bzz` url scheme assumes that the content referenced in the domain is a manifest and renders the content entry whose path matches the one in the request path. This offers the functionality of _virtual hosting_ of websites without dedicated webservers. Hosting on web3 is thus analogous to web2.0, with centralized hosting taken out of the equation.
+Manifests can also be mapped to a filesystem directory tree, which allows for uploading and downloading directories. Finally, manifests can also be considered indexes, so it can be used to implement a simple key-value store, or alternatively, a database index.
+
+In sum, the overall vision of swarm offers a fully decentralised peer-to-peer internet which is not only fault-tolerant, has zero downtime and censorship rexistance but also economically self-sustaining due to a built-in incentive system which, by compensating nodes for contributing their bandwidth and disk space achieve smooth low-latency scalable retrieval of popular content on the one hand and guarantees persistance of important yet rarely accessed data like archives or backups. For smooth delivery swarm will use the SWAP protocol while for storage guarantees it will use a two-tiered insurance system.
```

Comment:
censorship rexistance => censorship resistant

Reply:
which, by compensating nodes for contributing their bandwidth and disk space, achieves

Reply:
the one hand and guarantees persistance of important yet rarely accessed data like archives or backups
||
\\/
the one hand and guarantees persistence of important yet rarely accessed data like archives or backups on the other hand.